oob/constraints.lisp

```diff
@@ -878,13 +878,13 @@
   (call-to-LT 1 0 (prc-modexp-lead---ebs) 0 32))
 
 (defconstraint prc-modexp-lead---compare-ebs-and-cds (:guard (* (standing-hypothesis) (prc-hypothesis) (prc-modexp-lead-hypothesis)))
```

Comment:
Question: why do we have both `(prc-hypothesis)` and `(prc-modexp-lead-hypothesis)` ? Is that in the spec ?

Reply:
`(:guard (* (standing-hypothesis) (prc-modexp-lead-hypothesis)))` should be sufficient for all of these.